src/frontend/web_application/src/scenes/MessageList/components/Message/style.scss

```diff
@@ -1,160 +1,109 @@
   }
 
   &__top-bar {
-    @include flex-grid-row($size: expand);
+    @include flex-grid-row;
+    flex-wrap: nowrap;
     align-items: center;
-    border-bottom: 1px solid $co-color__fg__back--lower;
-    background: $co-color__fg__back;
+    padding: $co-component__spacing 0;
+    border-bottom: 1px solid $co-color__fg__border--higher;
+    color: $co-color__fg__text--high;
     font-size: $co-font__size--small;
-
-    &--is-unread {
-      background: $co-color__fg__back;
-    }
   }
 
-  &__with-attachements { @include flex-grid-column(shrink); }
   &__type { @include flex-grid-size(shrink); }
   &__type-icon { color: $co-color__fg__text; }
   &__type-label { display: none; }
 
-  &__pi { @include flex-grid-size(shrink); }
-
-  &__author {
-    @include flex-grid-column;
-    color: $co-color__fg__text--high;
-    font-weight: 600;
-  }
-
-  &__date {
-    @include flex-grid-column(shrink);
-    margin-left: auto;
-  }
-
-  &__actions-switcher {
+  &__participants {
     @include flex-grid-size(shrink);
-    border-left: 1px solid $co-color__fg__back--lower;
+    margin-left: auto;
   }
 
-  &__subject {
-    padding: $co-component__spacing;
-    border-bottom: 1px solid $co-color__fg__back--lower;
-    color: $co-color__fg__text--lower;
+  &__participant {
+    &::after { content: ', '}
 
-    &--is-unread {
-      background: $co-color__fg__back;
-      color: $co-color__fg__text;
+    &:last-child {
+      &::after { content: ''}
     }
   }
 
-  &__subject-checkbox {
+  &__date {
     @include flex-grid-size(shrink);
-    margin-left: auto;
   }
 
   &__content {
-    position: relative;
-    overflow: hidden;
-    color: $co-color__fg__text;
-    box-sizing: content-box;
-
-    &::before {
-      display: block;
-      content: '';
-      position: absolute;
-      bottom: 0;
-      left: 0;
-      width: 100%;
-      height: 0;
-      transition: height .1s linear;
-      background: linear-gradient(to bottom, transparent, transparent);
-    }
-
-    &--fold {
-      &::before {
-        height: $co-component__height;
-        background: linear-gradient(to bottom, transparent, $co-color__fg__back__low);
-      }
-    }
-  }
-
-  &__body {
-    padding: $co-component__spacing;
-    &--rich-text { @include o-reset-html-message; }
+    max-width: $message_content_width;
+    margin: 0 auto;
+    padding: $co-margin 0;
   }
 
-  &__footer {
-    min-height: $co-component__height;
-    background: $co-color__fg__back;
-    border-top: 1px solid $co-color__fg__back--lower;
-    color: $co-color__fg__text--lower;
-    line-height: $co-component__height;
-    text-align: center;
-  }
-
-  &__attachements-list {
-    padding-top: $co-margin;
-    border-top: 1px solid $co-color__fg__border--higher;
-  }
-
-  &__attachements-item {
-    @include flex-grid-row;
-    margin-bottom: $co-component__spacing;
-    background: $co-color__fg__border--higher;
-    font-size: $co-font__size--small;
+  &__subject {
+    display: block;
```

Comment:
why?

Reply:
`display: block` because of the 100% large border-bottom. Is it better to put a `div` instead of `span` for `m-message__subject` ? or maybe better a `h2` or any other title?

That make me think, we may better use an `<article>` tag to diplay each message
`<article id={message.message_id} className="m-message">`
and then `<header>` and `<footer>` tags.
What do you think?

Reply:
I think the more semantic the better
so :+1: 

http://devdocs.io/html/element/article